src/modules/Providers/views/migrate/components/PlansCreateForm.tsx

```diff
@@ -81,7 +82,7 @@ const buildNetworkMessages = (
   },
 });
 const buildStorageMessages = (
-  t: (key: string) => string,
+  t: TFunction,
```

Comment:
I think it is easier instead of adding a type, simply import t directly here and omit it from arg